app/org/sagebionetworks/bridge/models/ClientInfo.java

```diff
@@ -78,6 +79,16 @@ public ClientInfo load(String userAgent) throws Exception {
      * is still logged exactly as it is retrieved from the request.
      */
     public static final ClientInfo UNKNOWN_CLIENT = new ClientInfo.Builder().build();
+    
+    /**
+     * The osName that maps to an iOS client app.
+     */
+    public static final String IOS_OS_NAME = "iPhone OS";
+    
+    /**
+     * The osName that maps to an Android client app.
+     */
+    public static final String ANDROID_OS_NAME = "Android";
```

Comment:
I don't see where these are used or how you're addressing the space vs "/" issue... ?